packages/mds-ingest-service/repository/entities/telemetry-annotation-entity.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BigintTransformer, DesignType, IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { Timestamp, UUID } from '@mds-core/mds-types'
+import { Column, Entity, Index } from 'typeorm'
+@Entity('telemetry_annotations')
+export class TelemetryAnnotationEntity extends IdentityColumn(RecordedColumn(class {})) {
+  @Column('uuid', { primary: true })
```

Comment:
Is there supposed to be a `telemetry_row_id` or somesuch that points back at the original telemetry row id?

Reply:
Wasn't planning on it since that would require modifying the telemetry processor and a few other things to start passing around the `telemetry_id`, but I can put it in since people want it there.